extension/app/ts/components/subcomponents/icons.tsx

```diff
@@ -0,0 +1,25 @@
+export function ArrowIcon(param: { color: string }) {
+	return <>
+		&nbsp;
```

Comment:
Feels like we should be using styling here rather than `&nbsp;`?  e.g., `padding`

Reply:
Same below and I think I saw another one potentially above.